docs/docs/developers/guides/smart_contracts/writing_contracts/notes/custom_note.md

```diff
@@ -1,53 +0,0 @@
----
```

Comment:
Renamed this to implementing a note and largely rewrote that as it was very stale.